templates/service-template.yml

```diff
@@ -373,11 +373,6 @@ parameters:
   description: The public HTTP host URL of the service
   value: "https://api.openshift.com"
 
-- name: KAFKA_CAPACITY_INGRESS_THROUGHPUT
```

Comment:
Is this environment is not required? have we also added this in app-interface?

Reply:
We want to remove it, because those values will come from the new supported instance types array. However, i'm not sure if we should do the removal in the same PR.

@JameelB what do you think? Should we keep the existing env vars and default values at first?

Reply:
I think we can keep the existing env vars and default values for now and remove them in a separate PR since it's not only this one that will be removed. We won't need any of the `KAFKA_CAPACITY_*` once we start using the limits in the instance type/size configuration.